Add wallet migration widget to wallet migration page

samgutentag opened the pull request.

samgutentag committed:
add wallet migration widget to wallet migration page

samgutentag committed:
add wallet migration widget to wallet migration page

jthiller committed:
Help finding Hotspot queries

waveform06 committed:
Update validators-and-voting-power.mdx

Further clarifications, corrections and grammar updates.

waveform06 committed:
Update 5g-on-helium.mdx

Updated with new Emissions information and Solana dates.

waveform06 committed:
Update 5g-on-helium.mdx

Corrected mistake, 250B included 50B premine and 200B emissions

Sophi committed:
Create miner to gateway-rs configuration migration guide

jthiller committed:
prettier

jthiller committed:
stub out top nav

jthiller committed:
stubbing out more content and organization

jthiller committed:
prettier

jthiller committed:
Fixes Solana Migration sidebar

jthiller committed:
Add some detail to migration overview

Sophi committed:
**General Configuration (settings.toml)**

Co-authored-by: Marc Nijdam <marc@helium.com>

Sophi committed:
Capturing log output and writing to disk may be handled with a log manager such as journald or syslogd.

Co-authored-by: Marc Nijdam <marc@helium.com>

Sophi committed:
* Logging outputs (goes to stdout not to a file anymore)

Co-authored-by: Marc Nijdam <marc@helium.com>

Sophi committed:
Since gateway-rs no longer releases as an installable package it is up to the maker to integrate with their system startup setups, (like procd or tini, systemd depending on the operating system setup)

Co-authored-by: Marc Nijdam <marc@helium.com>

Sophi committed:
**Keypair configuration**

Co-authored-by: Marc Nijdam <marc@helium.com>

Sophi committed:
Update miner to gateway-rs configuration migration guide

Additional changes to draft

riobah committed:
Fixing the casing for HeliumGeek

jthiller committed:
Adds migration countdown backed by sentinel

samgutentag committed:
Apply suggestions from code review

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

jthiller committed:
stubs out architecture category

samgutentag committed:
formatting fixes

samgutentag committed:
line wrap

samgutentag committed:
emmisions table and link to HIP-75

samgutentag committed:
emissions table css

jthiller committed:
Stubs out tokens section

samgutentag committed:
corrected local link

samgutentag committed:
relocated emissions curve table

jthiller committed:
IoT network section

jthiller committed:
mobile network section

waveform06 committed:
Update helium-wallet-app.mdx (#1156)

* Update helium-wallet-app.mdx

Minor updates for v2.x.x

* Apply suggestions from code review

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* line wrap

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

jthiller committed:
Split out coverage mapping

jthiller committed:
keep announcementnav from wrapping

jthiller committed:
IoT section cleanup

jthiller committed:
More reorg

samgutentag committed:
swap html table for markdown table

samgutentag committed:
remove css for markdown table

jthiller committed:
Remove majority of WIP banners

jthiller committed:
Link to migration overview

jthiller committed:
Break out pages for API and ETL migration

ChewingGlass committed:
Ensure tx sending out sol always runs last. (#1167)

waveform06 committed:
Update wallet-user.mdx (#1166)

* Update wallet-user.mdx

Added link to Helium Wallet App page

* prettier

* minor copy edits, link to Hotspot App page

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>
Co-authored-by: Sam Gutentag <developer@samgutentag.com>

samgutentag committed:
Ledger Wallet Migration docs (#1153)

* links to github repo and notes on blind signing

* ledger migration tool and docs

jthiller reviewed (changes requested):
I think we should take the opportunity to do a few updates here.
- Should avoid the word 'seed' since that's somewhat sacred with seed phrase. 'Inflate' has been used in this context. 'Migrate' is probably most familiar.
- Would be good to give some confirmation. Currently the button just resets after Submitting.... Maybe we show account balances & Hotspots?

deasydoesit committed:
Adding file type and proto for foundation-iot-packet-ingest (#1169)

jthiller committed:
Reorder nav items (#1168)

jthiller committed:
Update migration counter (#1164)

* more time precision on countdown

* prettier

* Remove number padding and manage word wrapping

* Tidy

jthiller committed:
Updates Architecture/Solana (#1165)

Co-authored-by: Sam Gutentag <developer@samgutentag.com>

samgutentag committed:
remove double announcement header (#1170)

Sophi committed:
Update miner to gateway-rs configuration migration guide

added firewall rules and ports

samgutentag committed:
upgrade packages

Sophi committed:
Update maker-ethics.mdx (#1177)

* Update maker-ethics.mdx

Note about dual mining

* formatting fixup

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>

waveform06 committed:
Update hotspot-operator.mdx (#1175)

Added HNT will remain unchanged line

waveform06 committed:
Update wallet-user.mdx (#1176)

Added HNT and MOBILE will remain unchanged by this operation.

ChewingGlass committed:
Fix bug around creator address

riobah committed:
Updating HeliumGeek URL

riobah committed:
Updating HeliumGeek URL

dansku committed:
add hotspotty api documentation link

jthiller committed:
Updates two more references to Hotspotty API

mbthiery committed:
Add details on oracle bytestream reading (#1184)

* Add details on oracle bytestream reading

* Update docs/oracles/oracle-data.mdx

Co-authored-by: Marc Nijdam <marc@helium.com>

* Link directly to oracle lambda

---------

Co-authored-by: Marc Nijdam <marc@helium.com>

DanielChrobak committed:
Typo Fix (#1186)

* Typo fix

On line 53, previously was "You can either migrate you wallet using this widget:" and the second "you" was changed to "your" to now look like "You can either migrate you wallet using this widget:".

mbthiery committed:
Add read bytestream data example code  (#1188)

* Add details on oracle bytestream reading

* Add bytestream read examples

* Add code block filetype

* Add protobuf, rust, toml code theming

jthiller committed:
gateway-rs migration guide for makers

Co-Authored-By: Jeremy Cooper <25463150+ke6jjj@users.noreply.github.com>

jthiller committed:
Linting and formatting

samgutentag committed:
removing unused image files (#1173)

* removing unused image files

* restoring image files

dependabot[bot] committed:
Bump webpack from 5.75.0 to 5.76.3 (#1187)

Bumps [webpack](https://github.com/webpack/webpack) from 5.75.0 to 5.76.3.
- [Release notes](https://github.com/webpack/webpack/releases)
- [Commits](https://github.com/webpack/webpack/compare/v5.75.0...v5.76.3)

---
updated-dependencies:
- dependency-name: webpack
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

DeflateAwning committed:
Add python downlink example, minor cleaning (#1179)

jthiller committed:
OpenLNS - Running an LNS (#1171)

* Update Class C date.

Co-Authored-By: lytzV <44751312+lytzV@users.noreply.github.com>

* Init IoT Data Infrastructure docs

+ console operator migration
+ openLNS getting started
/ consolidate two open-lns overview docs

Co-Authored-By: lytzV <44751312+lytzV@users.noreply.github.com>

* Fix slug

Co-authored-by: Sam Gutentag <developer@samgutentag.com>

* prettier formatting

* prettier formatting sidebar

---------

Co-authored-by: lytzV <44751312+lytzV@users.noreply.github.com>
Co-authored-by: Sam Gutentag <developer@samgutentag.com>

ke6jjj committed:
Update gateway-rs configuration items and tables, especially keys. (#1190)

jthiller committed:
Starting to rework token pages.

jthiller committed:
MOBILE header img, mint address.

jthiller committed:
Edit pass on HNT token page.

jthiller committed:
Continued edits

jthiller committed:
Releasing before HIP 80

deasydoesit committed:
Update documentation for CLI wallet migration commands (#1194)

* Improve documentation for CLI wallet migration commands

* Updating based on pre-PR review

* Updating formatting

samgutentag committed:
github links, line wrap, few minor wording changes

deasydoesit committed:
Updates to better describe Helium CLI Wallet migration process (#1195)

* Improve documentation for CLI wallet migration commands

* Updating based on pre-PR review

* Updating formatting

* Updates to better describe Helium CLI Wallet migration process

* Prettier'd

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

Sophi committed:
Update maker-ethics.mdx (#1192)

error in #11, fixed wording

riobah committed:
Updating HeliumGeek URL

DanielChrobak committed:
Rewording (#1198)

I think we’ve been trying to use the word “reward” instead of “pay”

ChewingGlass committed:
Add better error messages (#1201)

Siegfried-B committed:
Update validators-and-voting-power.mdx (#1193)

* Update validators-and-voting-power.mdx

Correcting errors and suggesting improvements regarding: (a) HNT-veHNT positions, (b) DAO Utility Score

* line wrap formatting

* feedback

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>

ChewingGlass committed:
Fix solana signer block (#1202)

ChewingGlass committed:
Add instructions to sideload the ledger app (#1200)

* Add instructions to sideload the ledger app

* removing redirect

* Update docs/solana/migration/ledger.mdx

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

* formatting fixes

---------

Co-authored-by: Sam Gutentag <developer@samgutentag.com>
Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

waveform06 committed:
Update rewardable-entities.mdx (#1199)

ECC section spotted by user 👽 from Mars#5523 and notified on discord

samgutentag committed:
migration guide

samgutentag committed:
minor copy edits

DanielChrobak committed:
Grammar/Wording/Spelling Fixes (#1204)

* Grammar/Wording Change

* Grammar/Wording Change

* Grammar Change

* Grammar/Wording Changes

* Spelling Error Fix

* Fixing Bad Fix

* applying line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update network-user.mdx (#1205)

* Update network-user.mdx

Added post from lytz about no HNT to DC burn in console during migration

* line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

pankajrathore9599 committed:
Updated rewardable-entites.mdx file (#1189)

We don't have any function with `send` in the anchor, We have `sendAndConfirm` for executing the successful transaction in the anchor.

waveform06 committed:
Update mobile.mdx (#1206)

* Update mobile.mdx

Updated emissions schedule and references to other HIPs

* moving * to footnotes in table, line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update mobile.mdx (#1207)

* Update mobile.mdx

Added Year 1 starts 1st August 2022.

* line wrap

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update iot.mdx (#1208)

* Update iot.mdx

Updated HIP references and added a Year 1 date

* moving asterisks to footnotes

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

deasydoesit committed:
Updating oracle data page in anticipation of new data on 4/18 (#1209)

* Updating oracle data page in anticipation of new data on 4/18

* Updating based on pre PR review

ChewingGlass committed:
Fix tx ordering in ledger migration (#1210)

samgutentag committed:
language updates

ChewingGlass committed:
Use correct uri (#1211)

* Fix tx ordering in ledger migration

* -Fix url

* Use devnet

* add https

ChewingGlass committed:
Add video walkthough. (#1212)

* Add video walkthough.

* extra line

* adding admonition box

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Stubs out veHNT doc.

Should break this out into 3 docs eventually.

jthiller committed:
Update veHNT

jthiller committed:
Adding more of a narrative to migration...

...moving veHNT content to dedicated page.

jthiller committed:
Copy tweaks

ChewingGlass committed:
Remove channels instructions (#1214)

* Remove channels instructions

* updated youtube walkthrough link

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

Talitrus committed:
Helium-Solana Ledger app clarifications (#1215)

* reorder steps to install Helium-Solana app

Developer mode must be enabled first in order to show the Helium-Solana app

* Remove mention of GitHub for Helium-Solana app

* add Helium-Solana Ledger devnet warning

* Remove ref to uninstalling Solana Ledger app

* formatting adjustments

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
Maker Wallet SOL and DC (#1216)

* maker wallet SOL and DC needs

* nicer images and links to compression calculator

* typo

* feedback from review

* mint calculation tables

samgutentag committed:
link fixup (#1217)

samgutentag committed:
Update validator-operator.mdx (#1220)

Talitrus committed:
change 7d landrush to 10

samgutentag committed:
Hotspot App Version 5.0.0 docs updates (#1218)

* updated images

* Hotspot app docs redux

* formatting to maker apps page

* migration guide updates for new Hotspot app

jthiller committed:
Clean up Overview page (#1222)

* Update clock

* Clean up overview page. Add UTC time.

* More clock updates.

waveform06 committed:
Update validator-operator.mdx (#1223)

replaced some lost words.

samgutentag committed:
Helium Lorawan Roaming Start Guide (#1197)

* initial commit of Roaming quick start guide docs

* image updates and typo correction

Siegfried-B committed:
Update vehnt.mdx (#1224)

- Correcting lower limit for lock-up duration from 6 months to 1 day

jthiller committed:
Realms/Staking Docs (#1225)

* Staking with Realms
* Updates & linting

Siegfried-B committed:
Update realms.mdx

Fixing some typos

jthiller committed:
Linting

deasydoesit committed:
Migration guide updates for mainnet (#1226)

* Updates to exchange operator page

* Update blockchain node migration guide

* Updating based on JH feedback

openxzx committed:
Update default value to GW_API (#1219)

ChewingGlass committed:
Fix config (#1229)

waveform06 committed:
Update vehnt.mdx

Clarifications about veHNT value and the 6%

jthiller committed:
typos and clarification

jamiew committed:
Ledger migration requires Chrome/Brave/HID support (#1240)

Once again the people ignore the best browser of all time, Firefox

jthiller committed:
Explains low balance error.

vincenzospaghetti committed:
update iot premie docs (#1242)

* update iot premie docs

correction on active hotspots. this does not include data only hotspots

* data only premine note

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

ChewingGlass committed:
Add HST management widget (#1178)

* Add HST management widgget

* More docs

* missing .

* Update docs/solana/hst.mdx

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

* Update docs/solana/hst.mdx

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

* Fix dist

* Fix some ledger miration bugs

* Add colleciton key

* Change HST to our fanout impl

* Use real hst

* f

* Fix build

* Fix build

* dep

* fix build

* f

* does this fix

* try client only

* Rename to HST holder

---------

Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

jthiller committed:
Add landrush + lockup graphs

jamiew committed:
It's explorer-old not old-explorer

Might be worth setting up the other DNS record too

DeflateAwning committed:
Add spreading factor summary to frequency-plans page

ChewingGlass committed:
Add HNT to fee sol (#1244)

* Add HNT to fee sol

* Fix sidebar

* frontmatter, sidebar, and quick styling

* Small edits to copy

* Use separate svc

* Fixes double inclusion in sidebar.

* Try signAndSend

* Fix est

* Axios get

* decimal error

* Fix

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>
Co-authored-by: Abhay Kumar <75+abhay@users.noreply.github.com>

riobah committed:
Fixing the category for iot_reward_shares files (#1247)

waveform06 committed:
Update sol.mdx (#1260)

added line to say use the wallet app browser to open this page to get SOL in the wallet app.

Talitrus committed:
update SOL fee required for lockup (#1261)

* update SOL fee required for lockup

* lien wrap applied

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

deasydoesit committed:
Remove or update occurrences of 'on the Helium L1' (#1259)

* Remove or update occurrences of 'on the helium l1'

* Update docs/oracles/iot-proof-of-coverage-oracles.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* Update docs/oracles/iot-proof-of-coverage-oracles.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* applied line wrap

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

waveform06 committed:
Update helium-network.mdx (#1237)

* Update helium-network.mdx

updated with a questions shuffle and replacing HNT with IOT and removal of "Helium Blockchain"

* feedback from suggestions, line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update community-governance.mdx (#1236)

* Update community-governance.mdx

Minor text updates

* feedback from suggestions, line wrap, alphabetize lists

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update helium-hotspot-app.mdx (#1231)

* Update helium-hotspot-app.mdx

Changed from Approved Manufacturers to Manufacturers sell approved hotspots

* Update docs/mine-hnt/helium-hotspot-app.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

* link cleanup

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>
Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

waveform06 committed:
Update twelve-words.mdx (#1234)

* Update twelve-words.mdx

updated with wallet use depreciated text

* consolidating 12/24-word docs to “seed phrase” docs

link fixes
netlify redirects
page cleanup
“current phase” phrasing
sidebar updates

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update ledger.mdx (#1265)

Ledger firmware to be at least 2.1.0 requirement added

samgutentag committed:
hnt-sol swap video demo

jthiller committed:
Add Vote Multiplier to Realms doc. (#1262)

* Adds section on vote multiplier, plus other clarifications.

* Fix sidebar order for sol token

waveform06 committed:
Update maker-apps.mdx (#1233)

* Update maker-apps.mdx

several minor updates

* integrated feedback

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update helium-wallet-app.mdx (#1266)

quick temp update. Removed Solana preview content and updated ledger part and links

waveform06 committed:
Update vehnt.mdx (#1264)

* Update vehnt.mdx

minor text clarifications

* linewraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
Update docs/helium-tokens/sol.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

samgutentag committed:
note to close ledger live before starting migration (#1230)

* note to close ledger live before starting migration

* Update docs/solana/migration/ledger.mdx

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

---------

Co-authored-by: Joey Hiller <1965053+jthiller@users.noreply.github.com>

Ryan-Goldstein committed:
Switch MNTD/RAK App Links to Hotspotty Connect (#1272)

* Switch MNTD/RAK App Links to Hotspotty Connect

Official announcement from MNTD indicating that the MNTD app is not supported after the Solana migration and will be removed from app stores soon, and that Hotspotty Connect should be used moving forward: https://cdn.discordapp.com/attachments/404106811252408322/1099171159368544346/image.png

* Update maker-apps.mdx

adding an admonition to make sure this doesn't look like a broken link

* typo

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Clean up Price Oracle docs

jthiller committed:
Update links.

mbthiery committed:
Update ledger migration copy (#1277)

ke6jjj committed:
Add node about compressed NFT RPC requirement to use example.

jthiller committed:
Noting Helius as an option.

DanielChrobak committed:
Add a Solana Migration FAQ page to the docs (#1276)

* Transaction fee correction

Changed to match the now updated and corrected fee shown at https://docs.helium.com/helium-tokens/sol-token

* Create faq.mdx

* Delete faq.mdx

* Create migration

* Update and rename migration to migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update migration.mdx

* Update realms.mdx

* Rename migration.mdx to solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

* Update solana-migration.mdx

samgutentag committed:
remove $ to remove LaTex formatting, linewraps

waveform06 committed:
Update validators-and-voting-power.mdx (#1271)

* Update validators-and-voting-power.mdx

Updated and removed all references to validators.
Next pass to focus on voting and leave staking delegation to another doc
Can this be renamed to voting power.mdx but also have an alias of the previous name

* line wraps

* move page and create redirects

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

waveform06 committed:
Update realms.mdx (#1269)

* Update realms.mdx

Some additional clarifications

* rewording from feedback, line wraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

samgutentag committed:
removing cross link prefixes, line wraps (#1282)

samgutentag committed:
redirect (#1284)

jthiller committed:
Clarify delegations and epoch timing (#1286)

* Clarify delegations.

* Consistently capitalize 'landrush'

* Remove redundant 3x

samgutentag committed:
Removing Validator Pages (#1285)

* removing validator pages, images and sidebar sections

* sidebar cleanup

* removing cross links

* redirect for all docs goes to migration guide

* cutting out validator references

* more validator references removed

* restore files from feedback

* feedback revisions

* update explorer to explorer-old domain

* updating more explorer links

* move image file

ChristinaR04 committed:
Update maker-apps.mdx (#1289)

* Update maker-apps.mdx

Edits to include missing info and remove makers that are not MCC approved.

* Update maker-apps.mdx

* small cleanup

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

Sophi committed:
Update compliance-commitee.mdx (#1248)

* Update compliance-commitee.mdx

Updated links and dates

* Update compliance-commitee.mdx

fixed broken link

* linewraps

---------

Co-authored-by: Sam Gutentag <1404219+samgutentag@users.noreply.github.com>

jthiller committed:
Decaying vs Cliff

jthiller committed:
Stake Unlocking

jthiller committed:
Copy edits

samgutentag committed:
typo correction

samgutentag committed:
Merge branch 'feature/wallet-migration-widget' of https://github.com/helium/docs into feature/wallet-migration-widget

samgutentag commented:
what on earth is this... rebase sux
